packages/react-button/src/components/CompoundButton/useCompoundButtonClasses.ts

```diff
@@ -1,17 +1,24 @@
 import { makeVariantClasses, Theme } from '@fluentui/react-theme-provider';
 import { CompoundButtonState, CompoundButtonVariants } from './CompoundButton.types';
-import { useButtonClasses } from '../Button/useButtonClasses';
+import { buttonStyles, buttonVariants } from '../Button/useButtonClasses';
+import { ButtonVariants } from '../Button/Button.types';
+
+import { merge } from '@fluentui/utilities';
+import { concatStyleSets } from '@fluentui/style-utilities';
 
 const GlobalClassNames = {
   root: 'ms-CompoundButton',
   contentContainer: 'ms-CompoundButton-contentContainer',
   secondaryContent: 'ms-CompoundButton-secondaryContent',
 };
 
-export const useClasses = makeVariantClasses<CompoundButtonState, CompoundButtonVariants>({
+/**
+ * CompoundButton classes hook for mixing classes into a mutable state object.
+ */
+export const useCompoundButtonClasses = makeVariantClasses<CompoundButtonState, CompoundButtonVariants>({
   name: 'CompoundButton',
   prefix: '--button',
-  styles: {
+  styles: concatStyleSets(buttonStyles, {
```

Comment:
we should change `styles` and `variants` to support arrays, then do concatStyleSets / merge inside `makeVariantClasses`